jobs/release.yaml

```diff
@@ -90,10 +90,12 @@
           values:
             - 0
```

Comment:
this performs a snap upgrade from 1.xx/stable to 1.xx/candidate
* can the current stable snaps be upgraded to the potential NEXT stable snaps